### PR TITLE
wicked: enable aarch64 tests

### PR DESCRIPTION
**Pull request by asmorodskyi**
enable aarch64 tests

VR - 
https://openqa.suse.de/tests/3221297 - image creation
https://openqa.suse.de/tests/3224296 - usage of image

### CODE REVIEW

File: data/autoyast_sle12/autoyast_wicked_aarch64.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  <users config:type="list">
+    <user>
+      <fullname>Bernhard M. Wiedemann</fullname>
+      <encrypted config:type="boolean">false</encrypted>
+      <user_password>nots3cr3t</user_password>
+      <username>bernhard</username>
+    </user>
+    <user>
+      <encrypted config:type="boolean">false</encrypted>
+      <user_password>nots3cr3t</user_password>
```
Comment (cfconrad) [inline data/autoyast_sle12/autoyast_wicked_aarch64.xml]: Isn't this something we need to avoid?

Comment (asdil12) [reply data/autoyast_sle12/autoyast_wicked_aarch64.xml]: It is maybe not very nice that it is hardcoded - but it not a secret at all.
Also I wouldn't see a way to dynamically generate that files.
So I think it is ok.

File: data/autoyast_sle12/autoyast_wicked_aarch64.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  <bootloader>
+    <device_map config:type="list">
+      <device_map_entry>
+        <firmware>hd0</firmware>
+        <linux>/dev/vda</linux>
+      </device_map_entry>
+    </device_map>
+    <global>
+      <timeout config:type="integer">1</timeout>
+      <append>splash=verbose</append>
```
Comment (cfconrad) [inline data/autoyast_sle12/autoyast_wicked_aarch64.xml]: `autoyast_sle12/autoyast_wicked_x86_64.xml` has spaces around it. Maybe we should keep the files equal as possible.

File: data/autoyast_sle12/autoyast_wicked_aarch64.xml (hunk 1 above)
Comment (cfconrad) [reply data/autoyast_sle12/autoyast_wicked_aarch64.xml]: ok

Comment (okurz) [reply data/autoyast_sle12/autoyast_wicked_aarch64.xml]: There are quite some profile templates that use variable expansion. I think even two different approaches are used, e.g. see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/autoyast_sle15/mini.xml as well as https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/autoyast_qam/12_installation.xml.ep

Comment (asmorodskyi) [reply data/autoyast_sle12/autoyast_wicked_aarch64.xml]: was not aware about second one . will try it out during next round of working on this ...

**Pull request merged**